persistence/persistence-mybatis-test/src/test/java/org/cbioportal/persistence/mybatis/MutationalSignatureMyBatisRepositoryTest.java

```diff
@@ -0,0 +1,78 @@
+package org.cbioportal.persistence.mybatis;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.cbioportal.model.Mutation;
+import org.cbioportal.model.MutationCount;
+import org.cbioportal.model.SNPCount;
+import org.cbioportal.persistence.dto.AltCount;
+import org.cbioportal.persistence.dto.KeywordSampleCount;
+import org.cbioportal.persistence.dto.MutatedGeneSampleCount;
+import org.cbioportal.persistence.dto.SignificantlyMutatedGene;
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("/testContextDatabase.xml")
+@Configurable
+public class MutationalSignatureMyBatisRepositoryTest {
+
+	@Autowired
+    private MutationalSignatureMyBatisRepository mutationalSignatureMyBatisRepository;
+	
+	@Test
+	public void getSNPCountsByGeneticProfile() {
+		List<SNPCount> snpCounts = mutationalSignatureMyBatisRepository.getSNPCounts("study_tcga_pub_mutations");
+		
+		Assert.assertEquals(8, snpCounts.size());
+	}
+	
+	@Test
```

Comment:
It appears that none of the tests check the case where a query is made for a sample which does not exist. We should add that test to make sure no exception is thrown .. and it would be better to add a check where some of the requested sampleIds do match and some do not. (a mixed query) .. and make sure that the output is of the correct size.